### ISSUE TITLE
11.0 Planned breaking changes (not much!)

### ISSUE THREAD

**Issue by alexreardon** [atlassian/react-beautiful-dnd#1194]
Here is a list of planned breaking changes for `11.0`

### 1. React peer dependency

Moving from `^16.3.1` to `^16.8.0`. 

#### Reasons:

- Internally we will be moving things incrementally to hooks #871. 
- Need `16.8` to take advantage of `react-redux@7` https://github.com/reduxjs/react-redux/issues/1177
- Moving to `16.6` context API. Still using old school react context

#### 2.1 Name change

shouldRespectForce**Touch** => shouldRespectForce**Press**

We will be calling preventDefault on the following: 
`webkitmouseforcechanged` (mouse) and `touchforcechange` (touch)

I think moving to `shouldRespectForcePress` is a clearer name given that it can be fired from mouse inputs too. `ForcePress` communicates the intent better

#### 2.2 Flipping the default

See #1160. Opting for a better dragging experience out of the box

```diff
- shouldRespectForcePress: true,
+ shouldRespectForcePress: false,
```

**Comment by alexreardon** [comment]
A huge thanks to @markerikson for his hard work on `react-redux` 👏

**Comment by alexreardon** [comment]
Shipping #871

**Comment by alexreardon** [comment]
```bash
react-beautiful-dnd@11.0.0-beta
```

You can have a play!

**Comment by alexreardon** [comment]
`react-beautiful-dnd@11.0.0-beta.2` is out!

**Comment by alexreardon** [comment]
`react-redux@7` has gone stable. I hope to ship `11.0` soon 😲

**Comment by pwolaq** [comment]
Hi, I just want to report that #1138 is broken in `11.0.0-beta.2` - see here: https://codesandbox.io/s/ypv8xn70j

**Comment by alexreardon** [comment]
Thanks for the heads up @pwolaq - i will take a look

**Comment by alexreardon** [comment]
@pwolaq issue is now fixed in `11.0.0-beta.3`

**Comment by alexreardon** [comment]
Released: https://github.com/atlassian/react-beautiful-dnd/releases/tag/v11.0.0
